src/Compilers/CSharp/Portable/Emitter/Model/SymbolAdapter.cs

```diff
@@ -126,5 +126,23 @@ internal IEnumerable<CSharpAttributeData> GetCustomAttributesToEmit(ModuleCompil
                 }
             }
         }
+
+        internal Cci.TypeReferenceWithAttributes GetTypeRefWithAttributes(TypeSymbol type, Cci.ITypeReference typeRef)
```

Comment:
Would this be any simpler as a `ImmutableArray<Cci.ICustomAttribute> GetTypeRefAttributes(TypeSymbol type)` helper instead? Then callers would be: `yield return new Cci.TypeReferenceWithAttributes(typeRef, GetTypeRefAttributes(@interface));`.